backend-db/plugins/teams.js

```diff
@@ -0,0 +1,55 @@
+};
+
+module.exports = function () {
+    const lucius = new Lucius(this);
+
+    lucius.pluginInit(PLUGIN_NAME, next => {
+        db.ready().then(async () => {
+            await db.seedTeams(predefinedTeams());
+            next();
+        }).catch(e => next(e));
```

Comment:
This code block mixes async and promises, it should stick to one or the other.

All-promise alternative:

    lucius.pluginInit(PLUGIN_NAME, next => {
        db.ready()
        .then(db.seedTeams(predefinedTeams()))
        .then(next())
        .catch(e => next(e));
    });

All-async alternative:

    lucius.pluginInit(PLUGIN_NAME, async next => {
        try {
            await db.ready();
            await db.seedTeams(predefinedTeams());
            next();
        } catch (e) {
            next(e);
        }
    });